packages/oui-progress-tracker/README.md

```diff
@@ -19,27 +19,32 @@
       <span class="oui-progress-tracker__label">Choix du produit</span>
     </a>
   </li>
+  <li class="oui-progress-tracker__connector oui-progress-tracker__connector_complete"></li>
   <li class="oui-progress-tracker__step oui-progress-tracker__step_complete">
     <a href="#" class="oui-progress-tracker__status">
       <span class="oui-progress-tracker__label">Configuration du produit</span>
     </a>
   </li>
+  <li class="oui-progress-tracker__connector oui-progress-tracker__connector_complete"></li>
   <li class="oui-progress-tracker__step oui-progress-tracker__step_active">
     <a href="#" class="oui-progress-tracker__status">
       <span class="oui-progress-tracker__label">Ajout de produits complémentaires</span>
     </a>
   </li>
+  <li class="oui-progress-tracker__connector"></li>
   <li class="oui-progress-tracker__step">
     <span class="oui-progress-tracker__status">
       <span class="oui-progress-tracker__label">Options du produit</span>
     </span>
   </li>
+  <li class="oui-progress-tracker__connector"></li>
   <li class="oui-progress-tracker__step">
     <span class="oui-progress-tracker__status">
       <span class="oui-progress-tracker__label">Connexion</span>
     </span>
   </li>
-  <li class="oui-progress-tracker__step">
+  <li class="oui-progress-tracker__connector"></li>
+  <li class="oui-progress-tracker__step oui-progress-tracker__step_disabled">
     <span class="oui-progress-tracker__status">
       <span class="oui-progress-tracker__label">Paiement</span>
```

Comment:
This text appear in the margin

Reply:
The text overflow outside of the component is fixed :)